frontend/src/pages/books/BookDetail.tsx

```diff
@@ -565,44 +639,74 @@ export default function BookDetail() {
           <div className="flex col-12 justify-content-start p-1">
             <div className="flex p-0">
               <TextLabel label="Title:" />
-              <p className="p-component p-text-secondary text-900 text-3xl text-center my-0">
+              <p className="p-component p-text-secondary text-900 text-m text-center my-0">
                 {title}
+                <label className="text-blue-800">
+                  {remoteBook && remoteBook.title != title
+                    ? `\xa0 (Remote: ${remoteBook.title})`
```

Comment:
Can you leave a comment as to what the \xa0 means and this thing does.

Reply:
\xa0 adds a space, but I realized I can just use padding so I removed it